### PR TITLE
Updated description for kubeadm init --kubernetes-version

### PR DESCRIPTION
**Pull request by kad**
Update documentation based on code changes in kubernetes/kubernetes#41820

cc @luxas

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2593)

### CODE REVIEW

File: docs/admin/kubeadm.md [hunk 1]
```diff
@@ -143,10 +143,11 @@ By default, `kubeadm init` automatically generates the token used to initialise
 each new node. If you would like to manually specify this token, you can use the
 `--token` flag. The token must be of the format `<6 character string>.<16 character string>`.
 
-- `--use-kubernetes-version` (default 'v1.5.1') the kubernetes version to initialise
+- `--kubernetes-version` (default 'latest') the kubernetes version to initialise
 
 `kubeadm` was originally built for Kubernetes version **v1.4.0**, older versions are not
-supported. With this flag you can try any future version, e.g. **v1.6.0-beta.1**
+supported. Current version of `kubeadm` require at least **v1.6.0-alpha.3** due to enabled by default RBAC.
```
Comment (chenopis) [inline docs/admin/kubeadm.md]: Suggested grammar: "The current version of `kubeadm` requires at least **v1.6.0-alpha.3** due to RBAC being enabled by default."

**Comment by kad** [comment]
@chenopis thanks for review. updated.

**Comment by chenopis** [comment]
@kad Great, thanks!

**Comment by luxas** [comment]
Hmm, this shouldn't have been merged into master but the release-1.6 branch (do we have one?)

**Comment by chenopis** [comment]
@luxas Thanks for catching that. Yes, there is a `release-1.6` branch open that this can be merged into.

**Comment by luxas** [comment]
I think you should be able to cherrypick this to a new PR for the v1.6 branch, then it's okay for me :+1:

**Comment by chenopis** [comment]
@luxas Ok, thanks!

**Comment by chenopis** [comment]
Ok, this has been cherry picked and [merged](https://github.com/kubernetes/kubernetes.github.io/commit/7b17b51997894ea8d4a1e92cc42c6645ac49cb1b) into [`release-1.6`](https://github.com/kubernetes/kubernetes.github.io/commits/release-1.6).

**Pull request merged**